blizzard.lua

```diff
@@ -98,5 +99,10 @@ function oUF:DisableBlizzard(unit)
 		-- Blizzard_ArenaUI should not be loaded
 		Arena_LoadUI = function() end
 		SetCVar('showArenaEnemyFrames', '0', 'SHOW_ARENA_ENEMY_FRAMES_TEXT')
+	elseif(unit:match('(nameplate)%d*$') == 'nameplate') then
```

Comment:
why not just `elseif(unit:match('(nameplate)%d+$')) then`? Also + instead of * but that's wayne either way I suppose